salt/_pillar/metalk8s_private.py

```diff
@@ -1,8 +1,10 @@
 import os.path
 import logging
+import yaml
 
 
 SA_PRIVATE_KEY_PATH = "/etc/kubernetes/pki/sa.key"
+APISERVER_KEY_PATH = "/etc/metalk8s/apiserver.key"
```

Comment:
```suggestion
APISERVER_KEY_PATH = "/etc/metalk8s/crypt/apiserver.key"
```
So we have a clear distinction between PKI stuff and encryption stuff

Reply:
I was unsure about the path, good suggestion, done.